js/components/interface/query/customComponents/slideshowImageComponent.js

```diff
@@ -11,36 +11,82 @@ define(function (require) {
     constructor (props) {
       super(props);
 
-      this.state = { carouselFullyLoaded: false };
+      this.checkboxAction = this.checkboxAction.bind(this);
+      this.fireImageAction = this.fireImageAction.bind(this);
+      this.getImageInstanceVisibility = this.getImageInstanceVisibility.bind(this);
+
+      let initialCheckBoxState = this.getImageInstanceVisibility(this.props.rowData.id)
+
+      this.state = { carouselFullyLoaded: false, checked: initialCheckBoxState , imageID : '', imageInstanceLoading : false };
 
       this.isCarousel = false;
       this.imageContainerId = '';
       this.fullyLoaded = false;
     }
 
+    getImageInstanceVisibility (path) {
+      let initialCheckBoxState = false;
+      try {
+        let imageVariable = eval(path);
+        if (imageVariable !== undefined) {
+          initialCheckBoxState = imageVariable.isVisible();
+        }
+      } catch (e) { }
```

Comment:
@jrmartin What happens here? I am guessing this is surrounded by try because of the eval  - are we ok with doing nothing if the path doesn't exist? Maybe log into console that the path was not found?

Reply:
@gidili Yeah, that's it. I added a console.info statement here, not much else to do.